src/index.html

```diff
@@ -18,5 +22,94 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of a generic 'Company Logo', provide a more descriptive alternative text that describes the logo or the company it represents.